### ISSUE TITLE
How can I use responsive helpers?

### ISSUE THREAD

**Issue by viktorvan** [Dzoukr/Feliz.Bulma#40]
It feels like I am missing something when I am trying to use responsive helpers. I can't find them, or figure out how to conditionally hide/show content using media breakpoints.

https://bulma.io/documentation/modifiers/responsive-helpers/#hide

I expected them to be available on `helpers` so I could do something like this:

```fsharp
Bulma.column [
    helpers.isHiddenMobile
    prop.text "My column for larger screens only"
]
``` 

But there is no `helpers.isHiddenMobile`. Maybe there is another way to achieve what I am trying to do?

Of course, as workaround I can add the class with `prop.classes [ "is-hidden-mobile" ]`. I was just thinking there is a type-safe way to do it from within Feliz.Bulma.

**Comment by Dzoukr** [comment]
Hi @viktorvan,

it seems that I forgot to implement some helpers 😄 

This library is PR friendly so if you'll add them feel to share it back.

**Comment by Dzoukr** [comment]
Fixed in #64
